packages/flutter_tools/templates/package/LICENSE.tmpl

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Your Company. All rights reserved.
```

Comment:
we definitely don't want to be including any default licenses with any of the `flutter create` output